changelog.d/983.feature

```diff
@@ -0,0 +1 @@
+- Add tests for allowed symbols in user identifiers. Thanks to @bodqhrohro
```

Comment:
we don't keep a changelog for sytest, and we don't use newsfiles like this for it. Thank you for your diligence in adding one, but it's redundant here. Please could you remove it?